.eslintrc.js

```diff
@@ -25,11 +25,20 @@ try {
   const scanjsConfig = require('eslint-plugin-scanjs-rules');
   scanjsRulesConfig = {};
 
-  // bring in all the scanjs rules, prefixed with 'scanjs-rules',
+  // ScanJS rules we don't want to use:
+  const ignoredScanjsRules = [
+    // scanjs is looking for a different `connect` than the one we use often from react-redux
+    'call_connect',
```

Comment:
ran into some warnings due to this rule since I added the the use of `react-redux`'s `connect()` function